src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -401,20 +403,20 @@ public function has($id)
      */
     public function get($id, $invalidBehavior = ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE)
     {
+        $id = strtolower($id);
```

Comment:
Currently you get a deprecation error for each possible alias (by the looks of the code :angel:), so i tend to resolve aliases first and then logic of `get`. Consider it an early return.

Ill dig into this to expose the side effects.